src/LibVLCSharp.Forms/Platforms/Android/VideoViewRenderer.cs

```diff
@@ -53,6 +68,11 @@ protected override void OnElementChanged(ElementChangedEventArgs<VideoView> e)
 
         private void OnMediaPlayerChanging(object sender, MediaPlayerChangingEventArgs e)
         {
+            // Avoid updating the MediaPlayer if the Context from the Control is already disposed
```

Comment:
Did you actually encounter this case? The repro code and stacktrace you shared does not include these lines, so I wonder why the defensive check here?

Reply:
Yes, this happens each time, if you navigate using Shell. If you re-visit a page, the old render gets disposed and a new one is created.
You can check it here:
https://github.com/AndreasReitberger/LibVlcViewIsNullTest

Just run the test repo and switch from "About" to "About2" and back to "About". Then you see the exception. If you set a breakpoint on "OnMediaPlayerChanging" and "Dispose", you'll see when it's called.